### ISSUE TITLE
[Feature request] Long sentences.

### ISSUE THREAD

**Issue by HarleyQuinn-Z6** [coqui-ai/TTS#1333]
When I installed coqui-tts I was expecting to be able to do something like `tts < input.txt > output.wav`. But it turns out that the input text needs to be divided in short enough sentences or you get an error that says `Decoder stopped with max_decoder_steps`. There are examples of long utterances in the [ddc-samples](https://erogol.github.io/ddc-samples/) which made me believe it would accept any text. So until this issue is resolved, it could be mentioned in the docs something about the current limitations.

**Comment by thorstenMueller** [comment]
If you are sure, that the model you're using can deal with long sentences as input there's a simple way. Edit the config.json in the downloaded model directory and adjust the parameter `max_decoder_steps` to a higher value.

**Comment by HarleyQuinn-Z6** [comment]
I haven't seen any `config.json`, I see mostly python files. Where do I find it? I'm using the default model. 
Could it be that it needs a powerful computer to achieve the same results as the given examples? Because I'm using it in a virtual machine and it may be limited.

**Comment by thorstenMueller** [comment]
When i get you right you want just use an existing pretrained voice model. If it is so there's no need to clone the repo code. I guess you did a `pip3 install tts`. You'll find `config.json` and a checkpoint file in the directory that is printed above `/home/user/.local/share/tts/tts_models--en--ljspeech--tacotron2-DDC`. There should be a config.json in.

**Comment by HarleyQuinn-Z6** [comment]
It works fine like that. I guess I'm going to have to take a closer look at how this works to be able to use it correctly.

**Comment by StevenLi-phoenix** [comment]
System: Mac OS Monterey 12
NO GPU or cuda (only cpu

I used the default pip install tts installed model. And used the default model. But I am also getting a lot of warnings for max_decoder_steps.

I went to the /Users/***/Library/Application Support/tts/tts_models--en--ljspeech--tacotron2-DDC location and found the config.json file described in this question. But there is no option in the file to describe max_decoder_steps.

Could this be the wrong model or am I looking in the wrong location?
